src/index.js

```diff
@@ -30,6 +34,19 @@ function AddListItem() {
   }
 }
 
+function checkBox() {
+  const checkboxAll = document.querySelectorAll('.checkBox');
+  for (let i = 0; i < checkboxAll.length; i += 1) {
+    // eslint-disable-next-line no-loop-func
+    checkboxAll[i].addEventListener('click', () => { completeTask(checkboxAll[i].value, ListArray); });
+  }
+}
```

Comment:
- [ ] When a checked box is unchecked it doesn't update the `isCompleted` value. Kindly make it work. :pray: 

![Screenshot from 2021-09-24 02-11-23](https://user-images.githubusercontent.com/64894927/134577016-fb8bfd94-91dc-4e83-9778-c8b04c69f5eb.png)

Reply:
Hello thanks for your feedback I am not getting this problem working very good with me  here is the live demo 
https://mustabbas.github.io/TodoList/dist/

Reply:
Hi @mustabbas , can you please try to uncheck the already checked box and see if it changes the `isCompleted` value to false. :pray: